docs/testing.md

```diff
@@ -67,7 +68,18 @@ This just needs the path to the token file (with pre-pended slash), such as `"/t
 
 ### Testing with Travis-CI and Coveralls
 
-These tests are run (without credentials) automatically on a pull request to the Narrative Github repo. These are currently run through [Travis-CI](https://travis-ci.org/) and the coverage reported with [Coveralls](https://coveralls.io/). There should be nothing you need to do to make this work.
+These tests are run automatically on a pull request to the Narrative Github repo. These are currently run through [Travis-CI](https://travis-ci.org/) and the coverage reported with [Coveralls](https://coveralls.io/).
```

Comment:
Should this be github actions?

Reply:
Yes, and Codecov. Updated that paragraph and some other places in that document.